### ISSUE TITLE
Cancel button scope

### ISSUE THREAD

**Issue by irmela** [vigetlabs/active_material#42]
As part of `app/assets/stylesheets/active_material/components/buttons.scss` there is a rule applied to `.cancel a`. I think this is to general because in one of my projects we have a class `.cancel` on the body tag and so all links on the page are affected by this styling. I would suggest changing it to `.cancel > a`.

### Current
```css
.button,
.cancel a {
  @include am-btn;
}
```

### Improved
```css
.button,
.cancel > a {
  @include am-btn;
}
```

**Comment by nhunzaker** [comment]
Ah great, thank you!

**Comment by irmela** [comment]
Fixed by #43
